### DISCUSSION TITLE
Inspector shoud show a tip to show full property value on Resource-selected

### DISCUSSION THREAD

**Discussion by jzq740176597** [godotengine/godot-proposals#6638] [Editor]
### Describe the project you are working on

3d

### Describe the problem or limitation you are having in your project

https://i.postimg.cc/bNh1g0JK/4-5-2023-1-35-23-PM.jpg

Material resource selected, Resource path property is usually long and cliped by inspector-border.

I have to change width of inspector and recover back to SEE FULL. SHOULD show full value for any clipped prop **on hover**!

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Just show full value for any clipped prop **on hover**!

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

*

### If this enhancement will not be used often, can it be worked around with a few lines of script?

no

### Is there a reason why this should be core and not an add-on in the asset library?

core

**Comment by Calinou** [comment]
This should be relatively easy to do by setting the LineEdit's tooltip to the string's value. There should probably be a length limit for very long strings though (same for multiline text with TextEdit), so that a tooltip can't fill your entire screen.

**Reply by Calinou** [reply]
- Pull request opened: https://github.com/godotengine/godot/pull/76231
